### PR TITLE
Fix Kubeadm e2e Tests

### PR DESCRIPTION
**Pull request by jessicaochen**

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -8180,6 +8180,7 @@
       "--kubernetes-anywhere-kubelet-ci-version=latest",
       "--kubernetes-anywhere-kubernetes-version=ci/latest",
       "--provider=kubernetes-anywhere",
+      "--tag=latest",
```
Comment (krzyzacy) [inline jobs/config.json]: this `--tag` flag is not going to work for prow jobs..

Comment (jessicaochen) [reply jobs/config.json]: Will remove

**Comment by krzyzacy** [comment]
rerun a ci job is trickier, you need to go to places like http://prow.k8s.io/?type=periodic&job=periodic-kubernetes-e2e-kubeadm-gce-1-7 and hit rerun, and go into prow cluster to run the kubectl command.

**Comment by pipejakob** [comment]
As discussed offline, I believe all of the `--tag=latest` flags will be inert for our jobs, since we use kubernetes_e2e's LocalMode instead of DockerMode. Aside from that, LGTM as long as it works :-).

**Comment by krzyzacy** [comment]
--tag=latest only works for docker mode :-) you want to change to `e2e-kubeadm:latest` in prow/config.json

**Comment by krousey** [comment]
LGTM from my point of view after the tags are removed.

**Comment by krzyzacy** [comment]
/lgtm

either bump the image tag, or change that to `:latest` I guess

**Comment by jessicaochen** [comment]
@krzyzacy could you merge and bump the kubekins image to pickup the kubetest changes? I can bump the kubeadm image to pickup the changes afterwards (unless you feel like taking care of this as well).

**Comment by krzyzacy** [comment]
@jessicaochen you can simply `cd ./jenkins/e2e-images` and `make push` to get a new kubekins image.

**Comment by jessicaochen** [comment]
@krzyzacy 

Good to know bumping kubekins does not require special steps. I bumped the images. Look good for merge?

**Comment by krzyzacy** [comment]
:crossed_fingers:

**Pull request merged**